content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -39,10 +39,14 @@ kubectl. Complete documentation is found in the [kubectl manual](/docs/reference
 kubectl handles locating and authenticating to the API server. If you want to directly access the REST API with an http client like
 `curl` or `wget`, or a browser, there are multiple ways you can locate and authenticate against the API server:
 
- 1. Run kubectl in proxy mode (recommended). This method is recommended, since it uses the stored apiserver location and verifies the identity of the API server using a self-signed cert. No man-in-the-middle (MITM) attack is possible using this method.
- 1. Alternatively, you can provide the location and credentials directly to the http client. This works with client code that is confused by proxies. To protect against man in the middle attacks, you'll need to import a root cert into your browser.
+1. Run kubectl in proxy mode (recommended). This method is recommended, since it uses
```

Comment:
nit: This paragraph uses API server and then apiserver. To be consistent, does it make sense to use API server?
nit: Does it make sense to spell out certificate instead of cert?

Reply:
Done. Thanks for your suggestions.